docs/topics/assembly_creating-a-basic-vertx-application.adoc

```diff
@@ -13,10 +13,24 @@ In addition to xref:mission-rest-http-vertx[using a booster], you can create new
 include::proc_creating-an-application.adoc[leveloffset=+1]
 
 include::proc_deploying-an-application-to-openshift.adoc[leveloffset=+1]
+
+:parameter-runtime-name: {VertX}
+:parameter-group-name-runtime-plugin: io.reactiverse
+:parameter-artifact-name-runtime-plugin: vertx-maven-plugin
+:parameter-version-runtime-plugin: ${vertx-maven-plugin.version}
+:parameter-jar-file-name: my-app-fat.jar
+include::proc_deploying-an-application-to-standalone-rhel.adoc[leveloffset=+1]
+:parameter-jar-file-name:
```

Comment:
These should be undefined, i. e. `:parameter-jar-file-name!:`, not just set to an empty string. Same elsewhere.

Reply:
Yup, they certainly should be. My bad. Let me fix that. Thanks for the catch:)

Reply:
I have now undefined ```{parameter-jar-file-name}``` everywhere.